client/src/components/SearchResultsFilters.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { Box, Text, Heading, CheckBox } from 'grommet'
+import { getReadable } from '../helpers/readableNames'
+import { sortedObjectKeysByValues } from '../helpers/sortObjectKeys'
+import { useMaterialsSearch } from '../hooks/useMaterialsSearch'
+
+const isChecked = (queryFacets, facet) => {
```

Comment:
The performance of this could go from exp to linear as the facets increases. However as it stands now there won't be so many facets that it would be noticeable.